engine/hooks/trigger_analyses.go

```diff
@@ -167,8 +167,9 @@ func (s *Service) triggerAnalyses(ctx context.Context, hre *sdk.HookRepositoryEv
 						return err
 					}
 					hre.SignKey = apiAnalysis.Data.SignKeyID
-					hre.Username = apiAnalysis.Data.CDSUserName
```

Comment:
Should we update also scheduler_repository_event.go -> manageRepositoryEvent : to create initiator struct if it doesn't exist

Reply:
done